code/addons/docs/template/stories/docspage/error.stories.ts

```diff
@@ -0,0 +1,22 @@
+import { global as globalThis } from '@storybook/global';
+
+export default {
+  component: globalThis.Components.Button,
+  tags: ['autodocs'],
+  args: { label: 'Click Me!' },
+  parameters: { chromatic: { disable: true } },
+  decorators: [
+    // Skip errors if we are running in the test runner
+    (storyFn: any) =>
+      window?.navigator?.userAgent?.match(/StorybookTestRunner/) ? 'skip' : storyFn(),
```

Comment:
This doesn't seem to be working as intended in CI. @yannbf ?

Reply:
It's working for the bulk of the SBs, I don't quite get it.

Reply:
i guess it just means that string is an invalid return type for svelte stories? cc @JReinhold

Reply:
Hmm I thinks it's a deeper issue than that.

Svelte and Vue2 always calls `storyFn` from the inside out, passing the result to each decorator upwards. So even if you have a decorator that doesn't render anything within, the undecorated storyFn will already have been called, and the error will have been thrown.

https://github.com/storybookjs/storybook/blob/show-docs-errors/code/renderers/vue/src/decorateStory.ts/#L88

https://github.com/storybookjs/storybook/blob/show-docs-errors/code/renderers/svelte/src/decorators.ts/#L98

I'm unsure what to do about it here on the spot though.

Reply:
I can just change the story to not throw rather than use a decorator. This is a pretty weird edge case.